### DISCUSSION TITLE
hola

### DISCUSSION THREAD

**Discussion by jorgesniper** [Launch-X-Latam/MisionFrontEnd#577] [Team Posts]
hola buena noche, disculpen la molestia pero me inscribi con tiempo para lo de backend y me aparece en github que ya estoy dentro pero en la app de teams no me aparece alguien que me pudiera ayudar por favor

**Comment by Zenitram03** [comment]
https://teams.microsoft.com/l/team/19%3a59Tb8L_0PlM5uxQtC8AjVfcF-wvYCKBxsS1JRWZLnyo1%40thread.tacv2/conversations?groupId=e6804a94-ce40-41ca-a5c2-b287fc50036b&tenantId=4ae54b05-b77e-4224-aef1-8661422e0816
